src/jdk.jpackage/macosx/classes/jdk/jpackage/internal/resources/MacResources.properties

```diff
@@ -83,5 +84,7 @@ message.setfile.dmg=Setting custom icon on DMG file skipped because 'SetFile' ut
 message.install-dir-ignored=Warning: "--install-dir" option is ignored for DMG packaging. The installation directory will default to {0}.
 message.codesign.failed.reason.app.content="codesign" failed and additional application content was supplied via the "--app-content" parameter. Probably the additional content broke the integrity of the application bundle and caused the failure. Ensure content supplied via the "--app-content" parameter does not break the integrity of the application bundle, or add it in the post-processing step.
 message.codesign.failed.reason.xcode.tools=Possible reason for "codesign" failure is missing Xcode with command line developer tools. Install Xcode with command line developer tools to see if it resolves the problem.
+message.runtime-image-invalid=Provided runtime image at "{0}" is invalid or corrupted.
+message.runtime-image-invalid.advice=Runtime image should be valid JDK bundle or JDK image.
```

Comment:
We need negative test cases for these new error conditions.

Reply:
Test cases are added.

Reply:
Can you give a reference to a test that expects jpackage to fail with `message.runtime-image-invalid` error in the output? I can't find it.

Reply:
I just push it. See ErrorTest.java.